packages/teleport/src/services/apps/makeApps.ts

```diff
@@ -29,8 +29,11 @@ export default function makeApp(json: any): App {
     awsConsole = false,
   } = json;
```

Comment:
I understand that this wasn't done in this PR but the value here isn't `JSON` so the name is confusing. If you wouldn't mind updating it to be a bit more appropriate.

Reply:
this is the pattern we follow every where else so i am going to leave as is. i might be biased now since i'm so used to it, but to me it makes sense b/c json means `unformatted response from backend`

Reply:
ahh ok, but yeah that's not really clear to anyone coming from the outside as JSON is formatted, and a string :D